web_src/js/features/stopwatch.js

```diff
@@ -124,9 +126,8 @@ async function updateStopwatch() {
 
 function updateStopwatchData(data) {
   const watch = data[0];
-  const btnEl = document.querySelector('.active-stopwatch-trigger');
+  const btnEl = document.querySelector('.active-stopwatch');
```

Comment:
There are 2 elements now, why only `querySelector`?

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/30732/commits/9c0be40547733872153fba1926fb0fe50199c9fd.